package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "etna",
-  "version": "1.1.0-ui",
+  "version": "1.1.1-ui",
```

Comment:
One of my broad hopes for the monorepo is that it might make this kind of version bumping on every change unnecessary, since local dependencies could be cloned as siblings and built that way without the package manager intermediate.